packages/make-styles/src/renderer/createDOMRenderer.ts

```diff
@@ -57,9 +57,16 @@ export function createDOMRenderer(targetDocument: Document = document): MakeStyl
         const ruleCSS = rtl ? rtlCSS || css : css;
 
         renderer.insertionCache[cacheKey] = true;
-
-        (renderer.styleElement.sheet as CSSStyleSheet).insertRule(ruleCSS, renderer.index);
-        renderer.index++;
+        try {
+          renderer.styleElement.sheet?.insertRule(ruleCSS, renderer.index);
+          renderer.index++;
+        } catch (e) {
+          const regex = new RegExp(`:(${ignoreSuffixes})`);
```

Comment:
nit: can we move it into outer scope as `ignoreSuffixes`? And name it `ignoreSuffixesRegex`?